sceptre/aws-opendata/templates/opendata-bucket.j2

```diff
@@ -127,6 +136,27 @@ Resources:
               - !Sub arn:aws:s3:::${DataSetBucket}/*
     Type: AWS::S3::BucketPolicy
 
+  # Add owner file to the synapse bucket, requires the cloudformation S3 objects macro
+  # https://github.com/Sage-Bionetworks/aws-infra/tree/master/lambdas/cfn-s3objects-macro
+  {% if sceptre_user_data.SynapseIDs is defined %}
+  SynapseOwnerFile:
+    Type: AWS::S3::Object
+    Metadata:
+      cfn-lint:
+        config:
+          ignore_checks:
+            - E3001
+    Properties:
+      Target:
+        Bucket: !Ref Bucket
+        Key: owner.txt
+        ContentType: text
+      Body: >-
+  {%     for SynapseID in sceptre_user_data.SynapseIDs %}
+          {{ SynapseID }}
```

Comment:
Does this automatically add entries that are line separated?

For example if I had:
```
   sceptre_user_data:
     SynapseIDs:
       - "123"
       - "456
```

Would the resulting body of the owner.txt be:

```
123
456
```
?

Reply:
yes, precisely